source/deployment/sso-saml-ldapsync.rst

```diff
@@ -29,3 +29,20 @@ Once the synchronization with AD/LDAP is enabled, user attributes are synchroniz
     
     If a user is deactivated from SAML, their session won't expire until they're deactivated from AD/LDAP. However, they won't be able to log back in to Mattermost.
  
+Technical description of SAML synchronization with AD/LDAP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When enabled, SAML synchronization with AD/LDAP occurs in phases:
+
+1. Get all the current LDAP users from the Mattermost database who have ``Users.AuthService`` set to ``ldap``. This is a SQL query issued against the Mattermost database.
```

Comment:
Can we add what the SQL query is?

Reply:
`SELECT * FROM Users WHERE AuthService = :AuthService"`